core/trie/key.go

```diff
@@ -87,6 +91,7 @@ func (k *Key) String() string {
 	return fmt.Sprintf("(%d) %s", k.len, hex.EncodeToString(k.bitset[:]))
 }
 
+// also hard to understand
```

Comment:
It deletes the least significant bits of a Key

So if you have 0b1101 as your Key and call DeleteLSB(2) on it, it will become 0b11